### PR TITLE
Feature/handle accept language

### PR DESCRIPTION
**Pull request by katsuyasu-murata**
## Issues
Create a middleware that returns a hash including converted accept-language values in a request header.

## Fixes
* Fork [ring-clojure/ring-accept](https://github.com/ring-clojure/ring-accept)
* Fix regexes so that accept-language can convert
* Add tests
* Add Ciecle CI settings
* Fix README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,20 @@
-# Ring-Accept
+Ring-Accept-language
```
Comment (taurauko) [inline README.md]: I'd use markdown's H1
```suggestion
# Ring Accept-language
```

File: README.md [hunk 2]
```diff
@@ -1,13 +1,20 @@
...
+====
 
-A Clojure library designed to ... well, that part is up to you.
+## Overview
+
+Ring-Accept-language provides a middleware that sets a hash that convert an accept-language in a request header.
```
Comment (taurauko) [inline README.md]:
```suggestion
Ring-Accept-language provides a middleware that parses the Accept-language request header and injects the value into the request hash.
```

File: README.md [hunk 3]
```diff
@@ -1,13 +1,20 @@
...
 
 ## Usage
 
-FIXME
+Add a middleware after adding dependencies.
+
+    (defn some-middleware
```
Comment (taurauko) [inline README.md]: Wrap this in code with ```

File: README.md [hunk 4]
```diff
@@ -1,13 +1,20 @@
...
+      [handler]
+      (wrap-accept-language handler))
 
 ## License
 
-Copyright © 2014 James Reeves
+Copyright © 2019 Studist Corporation
 
 Distributed under the MIT License, the same as Ring.
```
Comment (taurauko) [inline README.md]: Confirm with company if MIT License is ok

File: README.md (hunk 1 above)
Comment (katsuyasu-murata) [reply README.md]: Thanks, I've fix it.

File: README.md (hunk 2 above)
Comment (katsuyasu-murata) [reply README.md]: Thanks, I've fix it.

File: README.md (hunk 3 above)
Comment (katsuyasu-murata) [reply README.md]: Thanks, I've fix it.

File: README.md (hunk 4 above)
Comment (katsuyasu-murata) [reply README.md]: Thank you for teaching me to write a license.
I've fixed it.

**Comment by katsuyasu-murata** [comment]
@taurauko 
Thank you for confirming fixes, and would you confirm points again?

File: project.clj [hunk 5]
```diff
@@ -1,11 +1,7 @@
-(defproject ring/ring-accept "0.1.0-SNAPSHOT"
-  :description "FIXME: write description"
+(defproject jp.studist/ring-accept-language "0.1.0-SNAPSHOT"
+  :description "Ring-Accept-language provides a middleware that parses the Accept-language request header and injects the value into the request hash."
   :url "http://example.com/FIXME"
```
Comment (taurauko) [inline project.clj]: You won't set an URL? studist.jp? biz.teachme.jp?

Comment (katsuyasu-murata) [reply project.clj]: according to reitit's project.clj, an url was set a Github url.
Therefore I've set Github url into an url key.

https://github.com/metosin/reitit/blob/8c9a7a42cb6ab9bb46535c604c3820019e1d88a1/project.clj#L3

**Review by taurauko — APPROVED** [review]

**Pull request merged**